src/components/PatchesPage/StatusSelector.tsx

```diff
@@ -56,9 +56,9 @@ const treeData = [
     key: PatchStatus.Started,
   },
   {
-    title: statusValToCopy[PatchStatus.Success],
-    value: PatchStatus.Success,
-    key: PatchStatus.Success,
+    title: statusValToCopy[PatchStatus.LegacySucceeded],
+    value: PatchStatus.LegacySucceeded,
+    key: PatchStatus.LegacySucceeded,
   },
```

Comment:
Does this mean that the filter won't work for newer patches with the `"success"` status? I think one workaround could to include both the legacy and new success statuses when we [query for the patches](https://github.com/evergreen-ci/spruce/blob/07fc78fe676d76b86ebdddf648c7769ae0b3a39a/src/pages/UserPatches.tsx#L37-L40)

Reply:
Handling both is going to be included on the backend! https://jira.mongodb.org/browse/EVG-19914?focusedCommentId=5626989&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5626989